reference/modules/modules.md

```diff
@@ -1,3 +1,4 @@
 # Modules
 
+* [Form flows](form-flow.md)
```

Comment:
Is it `Form flow` or `Form flows`?

Reply:
We use plural for everything else. While the feature itself is called form flow, the individual flows are also referred to as form flows/form flow instances/form flow definitions. Want me to change it to `Form flow` everywhere?

Reply:
Changed it to form flow almost everywhere.